pkg/graph/schema/types/model_to_operations_matrix.graphql

```diff
@@ -0,0 +1,95 @@
+# ModelsToOperationMatrix Notes:
+# - There's purposefully no `Milestones` query right now, since there's currently no need to query milestones oustide the scope of a Category
+type ModelsToOperationMatrix {
+  categories: [Category!]! # Categories is at the top level because you _can_ have empty categories
+  commonMilestones: [CommonMilestone!]!
+  solutions: [Solution!]! # Need to support solutions that _don't_ have milestones
+}
+
+# Milestone represents a Milestone that has been added to a Model Plan's MTO
+type Milestone {
+  # DB Fields
+  id: UUID!
+  name: String!
+  description: String! # Not sure if this is actually needed - I think there might only be descriptions on Milestones in the library? (CommonMilestone)
+  facilitatedBy: String # TODO: Make enum for `MilestoneFacilitator` (TBD if that's the same as `SolutionFacilitator`)
+  needBy: Time
+  status: String! # TODO: Make enum for `MilestoneStatus`
+  riskIndicator: String! # TODO: Make enum for `MTORiskIndicator`
+  isDraftMilestone: Boolean!
+
+  # Custom Resolvers
+  addedFromMilestoneLibrary: Boolean!
+  solutions: [Solution!]!
+  category: Category # Can be NULL if not part of a category (TODO: Unless we handle "Uncategorized" as an actual category)
+  # TODO Is ^^ confusing for fetching subcategories too? Should we just have 2 resolvers, one for `category`, one for `subcategory`?
+
+}
+
+# Solution represents a Solution that has been added to a Model Plan's MTO
+type Solution {
+  # DB Fields
+  id: UUID!
+  name: String!
+  description: String!
+  facilitatedBy: String # TODO: Make enum for `SolutionFacilitator` (TBD if that's the same as `MilestoneFacilitator`)
+  status: String! # TODO: Make enum for `SolutionStatus`
+  riskIndicator: String! # TODO: Make enum for `MTORiskIndicator`
+
+  # Custom Resolvers
+  relatedMilestones: [Milestone!]! # Can be empty, since you can have a solution with no milestones
+  addedFromMilestoneLibrary: Boolean! # TRUE if this is linked to a common milestone
```

Comment:
Is the bool above necessary? We probably would want to leave the link to the specific common solution, (though we could still implement this resolver if useful for the front end)

Reply:
Probably not - we will probably maintain that link as you said, but I don't think there's currently anything in the designs that would point towards needing to identify if a solution originated from the library or not. I'll nix it for now!

Reply:
56ab74e8